serv/website/sound.html

```diff
@@ -38,6 +38,108 @@
         </nav> <!-- END OF NAVBAR-EXPAND -->
     </header> <!-- END OF HEADER -->
     <main>
+        <div id="content-box" class="container-fluid">
+            <h2 class="text-uppercase">SOUND AROUND YOU<br> <span class="subheadercb">MERSEBURG</span></h2>
+            <div class="row">
+                <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xl-6">
+                    <div class="row row-cols-2"> <!-- row-cols-2 -->
+                        <img src="images/say3.jpeg" class="col img-fluid" alt="Workshop">
+                        <img src="images/say1.jpeg" class="col img-fluid" alt="Recording">
```

Comment:
we use .jpg and .png endings in other places on the website. i think we should strive to only use one raster image format and consistent naming. i suggest `.jpg`

Reply:
Feel free to change code and endings directly on your end to save time with re-upload of files on Google Drive. Sounds good?

Reply:
Done inside code. Please change the file endings on the actual files yourself.